adagucserverEC/Definitions.h

```diff
@@ -28,7 +28,7 @@
 #ifndef Definitions_H
 #define Definitions_H
 
-#define ADAGUCSERVER_VERSION "2.21.2" // Please also update in the Dockerfile to the same version
+#define ADAGUCSERVER_VERSION "2.21.3" // Please also update in the Dockerfile to the same version
```

Comment:
Big enough version bump for "breaking change" of EDR output?

Reply:
Changed to 2.22.0